### ISSUE TITLE
Dense forest doesn't appear on map correctly

### ISSUE THREAD

**Issue by victorxup** [We-the-People-civ4col-mod/Mod#939]
It looks like an ordinary Grassland tile without features and Huckleberries resource on it.
It it correct appearance?

![Screenshot_1](https://github.com/We-the-People-civ4col-mod/Mod/assets/23401480/142f8bae-8f3b-4fca-9995-c1b29f47f608)
![Screenshot_2](https://github.com/We-the-People-civ4col-mod/Mod/assets/23401480/643e20e8-e263-4781-bbef-832a1ab4ef9d)
![Screenshot_3](https://github.com/We-the-People-civ4col-mod/Mod/assets/23401480/a7e51281-798f-4cf1-981f-674021712f11)

**Comment by Schmiddie1979** [comment]
Hm, seems that there is nothing I can do so far. The bonus huckleberries works fine for me. Never seen this before. Maybe raystuttgart has an idea, because he added the bonus resources?

**Comment by victorxup** [comment]
> Hm, seems that there is nothing I can do so far. The bonus huckleberries works fine for me. Never seen this before. Maybe raystuttgart has an idea, because he added the bonus resources?

Could you please share a pic?
How it should look - huckleberries resource on dense forest on grassland terrain

**Comment by Schmiddie1979** [comment]
Have you started a new map or have you tried to set the bonus resource on dense forest with the Word Builder?. Does it look like that as well?

It should look like this - all 3 tiles have dense forest and huckleberries on it. 

![huckleberryondenseforest](https://github.com/We-the-People-civ4col-mod/Mod/assets/32595781/8c626e28-7ae6-4bf6-ac6b-06c6efc5a3cc)

**Comment by Schmiddie1979** [comment]
It should also not make any difference that a native village exists on the next tile:

![image](https://github.com/We-the-People-civ4col-mod/Mod/assets/32595781/b7cc57a1-d205-459a-b7ec-ad54cffe54c0)

**Comment by victorxup** [comment]
....aaaaand.....
I'm no more able to reproduce this issue.
It appears totally fine now.
![image](https://github.com/We-the-People-civ4col-mod/Mod/assets/23401480/37524d33-1355-47cc-b011-dec56e4bea74)

I don't think I'm able to provide any more useful info, so probably the issue can be closed

P.S. Didn't use WB, map was generated by FaireWeatherTweakEx

**Comment by raystuttgart** [comment]
Sometimes the graphics engine messes up visual display of Terrain Features a bit.
It may also depend on the way the graphic itself was created ... 

So yeah if somebody wants to redo the Dense Forrest, he can try.
But the issue is so rare and minor ... i.e. just a small random visual bug that it is hardly worth the time.

Removing and replacing the Terrain Feature usually fixes it though.
So if you guys want to keep this open ok.

But I doubt anybody will invest hours for try and error to change anything.
In fact doing so is much more likely to break something than to fix it.

Those 3D Terrain graphics are not exactly "perfect programming science". ;)

**In other words:**
I suggest to close the issue and accept it as it is.
Alternatively someboy can try to completely redo it.

**Comment by victorxup** [comment]
Encountered it again. Dense Forest not so dense.
Seems that now It is displayed only partially - the space near the river is plain.
I suppose this indeed is related to river proximity
![not so dense forest](https://github.com/We-the-People-civ4col-mod/Mod/assets/23401480/65d1dd8d-932b-420c-8c80-84d3b339bf46)

**Comment by Nightinggale** [comment]
If you can provide a savegame where it happens consistently, then maybe we will have a chance to figure out what is going on.

**Comment by victorxup** [comment]
Tile x74:y48
[not_so_dense_forest.zip](https://github.com/We-the-People-civ4col-mod/Mod/files/13434811/not_so_dense_forest.zip)
